src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -202,6 +202,8 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
         Object.keys(this.attributes).forEach((key) => {
             widget.setConfig(key, configValues[key].value);
         });
+
+        widget.fireEditorInteraction('change');
```

Comment:
While firing the `editorInteraction` event is the proper fix, I'm not sure if the update button should be responsible for that.
I think it would have more sense if the `widget` fires the event itself when something was changed like it does when the widget is moved. So I think this event should be fired after the `setName`, `setWidgetAttributes`, `setAlignment`, `unsetAlignment`, `clearConfig` and 
`setConfig` methods are invoked. Each event firing should be of course debounced to prevent firing the `setConfigEvent` too many times.

Reply:
Thanks @dew326 for the suggestion!